webpack.config.js

```diff
@@ -16,23 +16,29 @@
 
 'use strict';
 
+const fs = require('fs');
```

Comment:
According to fs-extra's documentation, it copies all of fs's methods to itself, so you probably don't need to require this?

Reply:
Done